src/context/Theme.tsx

```diff
@@ -30,10 +30,9 @@ export type Variables =
     | "tertiary-foreground"
     | "tooltip"
     | "status-online"
-    | "status-away"
```

Comment:
While revite is in maintenance mode, it seems pointless to break existing themes.

Reply:
bump